src/psyclone/tests/domain/common/transformations/alg_invoke_2_psy_call_trans_test.py

```diff
@@ -191,8 +192,8 @@ def test_ai2psycall_apply_error(fortran_reader):
     with pytest.raises(TypeError) as info:
         trans.apply(invoke)
     assert ("Expected Algorithm-layer kernel arguments to be a literal, "
```

Comment:
Oh, I see that we explicitly don't support expressions (I'd forgotten that) as arguments so you can disregard my previous comment.